### DISCUSSION TITLE
Default colors for preview-tui/iconlookup should be read from $NNN_FCOLORS

### DISCUSSION THREAD

**Discussion by raj-magesh** [jarun/nnn#1671] [Ideas]
By default, the icon (and text) colors when listing directories/files in the preview-tui pane are the default nnn colors.

A better default would be to read the colors in $NNN_FCOLORS.

Thoughts?

(Aside: why is NNN_FCOLORS limited to those few filetypes instead of all the filetypes in .iconlookup?)

**Comment by N-R-K** [comment]
Icon colors are determined at compile time. You can edit `src/icons.h` if you want to change the icon colors.

> (Aside: why is NNN_FCOLORS limited to those few filetypes instead of all the filetypes in .iconlookup?)

Icons work based on file extension. NNN_FCOLOR is for the actual filetype.

**Reply by raj-magesh** [reply]
Sorry, I realize I wasn't clear. I meant that NNN_COLORS and NNN_FCOLORS don't seem to apply to the preview-tui pane when it's previewing the contents of a directory. So the colors in my nnn pane and the preview pane are very different. It's not just the icons, it's the text too.

**Reply by N-R-K** [reply]
@luukvbaal can answer that perhaps.
